lib/mayaUsd/ufe/UsdBatchOpsHandler.cpp

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright 2022 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "UsdBatchOpsHandler.h"
+
+#include <mayaUsd/ufe/UsdUndoDuplicateSelectionCommand.h>
+
+PXR_NAMESPACE_USING_DIRECTIVE
```

Comment:
Do you need this? All new code should use the fully qualified namespace name. But I don't see any USD code here.